### PR TITLE
NOISSUE - Add system run script

### PR DESCRIPTION
**Pull request by drasko**
Signed-off-by: drasko <drasko.draskovic@gmail.com>

### CODE REVIEW

**Review by dborovcanin — DISMISSED** [review]

File: scripts/run.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# Mainflux
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+###
+# Runs all Mainflux microservices (must be previously built and installed).
+#
+# Expects that PostgreSQL and needed messaging DB are alredy running.
+# Also, do not forget to run `cd mqtt && npm install` previously.
```
Comment (anovakovic01) [inline scripts/run.sh]: Maybe add this `npm install` to `Makefile`.

File: scripts/run.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+#
+###
+
+BUILD_DIR=../build
+
+# Kill all mainflux-* stuff
+function cleanup {
+	pkill mainflux
+    pkill nats
```
Comment (anovakovic01) [inline scripts/run.sh]: Fix alignment.

File: scripts/run.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+# Switch to top dir to find *.proto stuff when running MQTT broker
+cd ..
+MF_THINGS_URL=localhost:8183 node mqtt/mqtt.js &
+cd -
+
+###
+# CoAP
+###
+
+
```
Comment (anovakovic01) [inline scripts/run.sh]: Remove one empty line.

File: scripts/run.sh [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+# Users
+###
+$BUILD_DIR/mainflux-users &
+
+###
+# Things
+###
+MF_THINGS_HTTP_PORT=8182 MF_THINGS_GRPC_PORT=8183 $BUILD_DIR/mainflux-things &
+
+
```
Comment (anovakovic01) [inline scripts/run.sh]: Remove one empty line.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

File: scripts/run.sh (hunk 1 above)
Comment (drasko) [reply scripts/run.sh]: Good idea, let me try :)

**Review by anovakovic01 — DISMISSED** [review]
LGTM!

**Pull request merged**